var/spack/repos/builtin/packages/freeimage/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
```

Comment:
`os` unused?

Reply:
Not needed anymore. In the end, I solved the installation issues using a patch.